bukkit/src/main/java/com/leonardobishop/quests/bukkit/command/QuestsCommandSwitcher.java

```diff
@@ -76,21 +77,16 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
 
     @Override
     public void showHelp(CommandSender sender) {
-        sender.sendMessage(ChatColor.GRAY.toString() + ChatColor.STRIKETHROUGH + "------------=[" + ChatColor.RED + " Quests v" + plugin
-                .getDescription().getVersion() + " " + ChatColor.GRAY.toString() + ChatColor.STRIKETHROUGH + "]=------------");
-        sender.sendMessage(ChatColor.GRAY + "The following commands are available: ");
-        sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests " + ChatColor.DARK_GRAY + ": show quests");
-        sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests c/category <categoryid> " + ChatColor.DARK_GRAY + ": open category by ID");
-        sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests q/quest <questid> (start|cancel|track) " + ChatColor.DARK_GRAY + ": start, cancel or track quest by ID");
-        sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests started " + ChatColor.DARK_GRAY + ": show started quests");
-        if (sender.hasPermission(subcommands.get("random").getPermission())) {
-            sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests random " + ChatColor.DARK_GRAY + ": show random quests");
-        }
-        if (sender.hasPermission(subcommands.get("admin").getPermission())) {
-            sender.sendMessage(ChatColor.DARK_GRAY + " * " + ChatColor.RED + "/quests a/admin " + ChatColor.DARK_GRAY + ": view help for admins");
+        Set<String> permissions = plugin.getConfig().getConfigurationSection("command.player-help").getKeys(false);
+        for (String permission : permissions) {
+            if(!permission.contains("default") && !sender.hasPermission(subcommands.get(permission).getPermission())) continue;
+            for(String message : plugin.getConfig().getStringList("command.player-help."+permission)) {
```

Comment:
Please don't use a string list to define multiline messages, instead use on of the many different ways to define newlines in YAML 

https://yaml-multiline.info/